external/common/include/fast_cpp_csv_parser/README.md

```diff
@@ -0,0 +1,268 @@
+# Fast C++ CSV Parser
```

Comment:
I was thinking just `#include <csv.h>` or whatever but if you want to leave it in the subdir that's fine with me.